### PR TITLE
Starting with Scheduler + big refactoring

### PR DESCRIPTION
**Pull request by attila0x2A**

### CODE REVIEW

**Comment by RomanTymchyshyn** [comment]
@googlebot I consent.

File: commands.go [hunk 1]
```diff
@@ -74,15 +70,23 @@ func init() {
...
+)
 
 type CallbackAction int
 
 const (
 	SaveWordAction CallbackAction = iota
 	PracticeKnowAction
 	PracticeDontKnowAction
-	ChangeSettingAction
+	PracticeDontKnowActionNoPractice
```
Comment (RomanTymchyshyn) [inline commands.go:89]: Hard to figure out what is this for without looking at the code.
Wouldn't PracticeResetProgress be more descriptive?

File: settings.go [hunk 2]
```diff
@@ -30,6 +30,9 @@ type Settings struct {
 	// which to accept the translations.
 	// true if translation is accepted
 	TranslationLanguages map[string]bool
+	// TimeZone should be from IANA time zone database, e. g. America/New_York
```
Comment (RomanTymchyshyn) [inline settings.go:33]: The comment is no longer relevant. Time zones are in UTC+XX format

**Review by RomanTymchyshyn — APPROVED** [review]

File: commands.go (hunk 1 above)
Comment (attila0x2A) [reply commands.go:89]: Мені лінь зараз оновляти, то я поки так залишу, але потім можна й змінити :)

**Comment by attila0x2A** [comment]
Superseded by https://github.com/Uran198/words-telegram-bot/pull/34

**Pull request closed without merging**